identity/src/main/java/com/android/identity/SessionEncryptionDevice.java

```diff
@@ -146,7 +146,7 @@ public SessionEncryptionDevice(@NonNull PrivateKey eDeviceKeyPrivate,
             mapBuilder.put("data", messageCiphertextAndAuthTag);
         }
         if (statusCode.isPresent()) {
-            mapBuilder.put("status", statusCode.getAsInt());
+            mapBuilder.put("status", statusCode.getAsLong());
```

Comment:
I would have suggested to use longExact() in a follow-up commit, except that it was only introduced in API 31. Perhaps we should introduce a private conversion method that checks for whether the BigInteger fits into a long?

Same in line 215.

Reply:
Yeah, I'm going to add a helper (after this change is merged)